### PR TITLE
Fix data for pending pools

### PR DESCRIPTION
**Pull request by veado**
This PR fixes missing pending pools in pools overview

### CODE REVIEW

File: src/renderer/services/midgard/pools.ts [hunk 1]
```diff
@@ -145,6 +145,11 @@ const createPoolsService = (
    * Ddata of pending `Pools` from Midgard
    */
   const apiGetPoolsPending$: LiveData<Error, PoolDetails> = apiGetPools$({ status: GetPoolsStatusEnum.Staged })
+  /**
+   * Data of all `Pools` or by given status
+   */
+  const apiGetPoolsByStatus$: (status?: GetPoolsStatusEnum) => LiveData<Error, PoolDetails> = (status) =>
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/services/midgard/pools.ts:151]: just nitpick
`PoolDetailsLD`

File: src/renderer/services/midgard/pools.ts [hunk 2]
```diff
@@ -145,6 +145,11 @@ const createPoolsService = (
...
+    apiGetPools$({ status })
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/services/midgard/pools.ts:152]: and just to clarify. why do we need this ? it only calls `apiGetPools$` factory with `{status}`. is not it ?

**Review by thatStrangeGuyThorchain — APPROVED** [review]
just a question. merge it if you wish

File: src/renderer/services/midgard/pools.ts (hunk 2 above)
Comment (veado) [reply src/renderer/services/midgard/pools.ts:152]: If `status = undefined`, data of all pools (pending + availabe) are requested

Comment (thatStrangeGuyThorchain) [reply src/renderer/services/midgard/pools.ts:152]: ok

**Review by thatStrangeGuyThorchain — APPROVED** [review]
LGTM!

File: src/renderer/services/midgard/pools.ts (hunk 2 above)
Comment (veado) [reply src/renderer/services/midgard/pools.ts]: [Last commit](https://github.com/thorchain/asgardex-electron/pull/833/commits/4119783bbe59d79085800b8d5e4a1e457fd24353) has improved it ^

**Pull request merged**